### ISSUE TITLE
[SCREENREADER]: Improve the usability of our pagination links

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/veteran-facing-services-tools#155]
## Issue
The page numbers are not announcing the current page to assistive device users. They're also not announcing themselves as links because we don't have an `href` attribute on each link. I'd like to improve these links by refactoring the markup a bit. Screenshot and code example attached below.

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/713

## Acceptance Criteria
* As a screen reader user, I want the current page of results to announce as the current item.
* I also want the other page result links to announce themselves as links.

## Environment
* Windows 10
* IE11 + JAWS
* https://staging.va.gov/find-locations/ (Searched for "Salt Lake City", Community Care, Nurse Practitioner)

## Possible Fixes (optional)
The HTML example below is truncated, but should offer a good outline of what we'd like to accomplish for the pagination links.

```html
<span class="...">
  <a href="#" class="va-button-link" id="first-click" aria-label="First page">
    <span class="...">First</span>
    </span>
  </a>
</span>

<span class="va-pagination-prev">
  <a href="#" class="va-button-link" id="pager-previous-click" aria-label="Load previous page">Prev</a>
</span>

<!-- I'd like to add real URLs so the pages announce as links -->
<div class="va-pagination-inner">
  <ul>
    <li>
      <a aria-current="true" aria-label="Load page 2" class="va-pagination-active" href="real-href-of-links">2</a>
    </li>
    <li>
      <a aria-label="Load page 2" class="va-pagination-active" href="real-href-of-links">2</a>
    </li>
    <li>
      <a aria-label="Load page 3" class="va-pagination" href="real-href-of-links">3</a>
    </li>
  </ul>
</div>

...
<!-- Next link should be spelled out, and Last link too -->
```

## WCAG or Vendor Guidance (optional)
* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)
* [Using the aria-current attribute - Tink.UK](https://tink.uk/using-the-aria-current-attribute/)

## Screenshots or Trace Logs
<img width="964" alt="Screen Shot 2019-08-16 at 3 26 48 PM" src="https://user-images.githubusercontent.com/934879/63196187-4c2a7d80-c03a-11e9-8f3c-3e816c3ee2c3.png">

**Comment by alexpappasoddball** [comment]
@1Copenut Did you intentionally leave the triage tags off this one?  I noticed you added me as an assignee, just wanted to confirm if it belongs with the us (triage team).
